policyengine_us/variables/gov/states/ut/tax/income/credits/ut_at_home_parent_credit.py

```diff
@@ -7,35 +7,22 @@ class ut_at_home_parent_credit(Variable):
     label = "Utah at-home parent credit"
     unit = USD
     definition_period = YEAR
-    defined_for = StateCode.UT
+    defined_for = "ut_at_home_parent_credit_agi_eligible"
+    reference = (
+        "https://le.utah.gov/xcode/Title59/Chapter10/59-10-S1005.html",
+        "https://www.taxformfinder.org/forms/2021/2021-utah-tc-40-full-packet.pdf#page=23",
+    )
 
     def formula(tax_unit, period, parameters):
         person = tax_unit.members
         age = person("age", period)
         is_dependent = person("is_tax_unit_dependent", period)
         p = parameters(period).gov.states.ut.tax.income.credits.at_home_parent
         count_children = tax_unit.sum((age < p.max_child_age) & is_dependent)
-        parent_qualifies = (
-            person("irs_employment_income", period)
-            + person("self_employment_income", period)
-        ) < p.parent_max_earnings
-        one_parent_qualifies = tax_unit.sum(parent_qualifies) > 0
-        tax_unit_qualifies = (
-            tax_unit("adjusted_gross_income", period) < p.max_agi
+
+        one_parent_income_eligible = person(
+            "ut_at_home_parent_credit_earned_income_eligible_person", period
         )
-        max_credit = (
-            p.amount
-            * count_children
-            * one_parent_qualifies
-            * tax_unit_qualifies
+        return tax_unit.sum(
+            p.amount * count_children * one_parent_income_eligible
```

Comment:
```suggestion
        individual_credit_amount = p.amount * count_children * one_parent_income_eligible
        return tax_unit.sum(
            individual_credit_amount
```

Reply:
just to confirm, if two parents are eligible and they have 2 kids they get 4x the amount?

Reply:
https://chat.openai.com/share/d6d99f81-13ab-4301-a7e1-a6cf2a0f1638

> (2) For a taxable year beginning on or after January 1, 2000, a claimant may claim on the claimant's individual income tax return a nonrefundable tax credit of $100 for each qualifying child if:
(a) the claimant or another claimant filing a joint individual income tax return with the claimant is an at-home parent; and 
(b) the adjusted gross income of all of the claimants filing the individual income tax return is less than or equal to $50,000.